runtime/process.go

```diff
@@ -24,15 +29,49 @@ func NewPHPProcess(args ...string) *PHPProcess {
 	}
 }
 
+func (proc PHPProcess) vendorDir(workingDir string) (string, error) {
+	tmpDir := path.Join(workingDir, "vendor", "_laravel-ls")
+	if err := os.MkdirAll(tmpDir, 0o755); err != nil {
+		return "", err
+	}
+	return tmpDir, nil
+}
+
 // Exec executes the PHP code in the specified working directory.
 // It returns an io.Reader with the output or an error if execution fails.
 func (proc PHPProcess) Exec(workingDir string, code []byte) (io.Reader, error) {
 	outBuf := &bytes.Buffer{}
 	errBuf := &strings.Builder{}
 
-	// Prepare the command with the PHP binary and code as arguments.
-	// proc.Args[0] is the PHP binary (e.g., "php"), and the rest are additional arguments.
-	cmd := exec.Command(proc.Args[0], append(proc.Args[1:], string(code))...)
+	vendorDir, err := proc.vendorDir(workingDir)
+	if err != nil {
+		return nil, err
+	}
+	
+	// hash code content for temporary file name
+	hash := md5.New()
+	hash.Write(code)
+	hashValue := hash.Sum(nil)
+	filePath := path.Join(vendorDir, fmt.Sprintf("laravel-ls-%x.php", hashValue))
```

Comment:
This can be simplified.

```suggestion
	filePath := path.Join(vendorDir, fmt.Sprintf("laravel-ls-%x.php", md5.Sum(code)))
```